### PR TITLE
Documentation patches

### PR DESCRIPTION
**Pull request by EvgenyAlterman**

### CODE REVIEW

File: documentation/docs/installation.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ You will need an `.env` file to run it. We will configure it in the next step.
 To access Quix, navigate to:  
 `http://localhost:3000`
 
+## Clone Repo
```
Comment (antonpodolsky) [inline documentation/docs/installation.md:24]: Clone Repo => Clone Repository

File: documentation/docs/installation.md [hunk 2]
```diff
@@ -21,6 +21,11 @@ You will need an `.env` file to run it. We will configure it in the next step.
...
+Clone Git Repository
```
Comment (antonpodolsky) [inline documentation/docs/installation.md:25]: can remove this I think

**Review by antonpodolsky — CHANGES_REQUESTED** [review]

File: documentation/docs/installation.md (hunk 2 above)
Comment (EvgenyAlterman) [reply documentation/docs/installation.md:25]: leave without any title? or just subtitle

Comment (antonpodolsky) [reply documentation/docs/installation.md:25]: Leave just `## Clone repository`

**Review by antonpodolsky — APPROVED** [review]

**Comment by antonpodolsky** [comment]
@EvgenyAlterman you can merge it.

**Pull request merged**